src/vellum/workflows/nodes/displayable/code_execution_node/utils.py

```diff
@@ -10,3 +15,81 @@ def read_file_from_path(node_filepath: str, script_filepath: str) -> Union[str,
         with open(full_filepath) as file:
             return file.read()
     return None
+
+
+class ListWrapper(list):
```

Comment:
Copied most of this from: https://github.com/vellum-ai/code-executor-service/blob/main/server/sandboxes/python/runner.py